README.adoc

```diff
@@ -52,6 +52,26 @@ And then you can deploy the subgraph. Follow the CLI instructions:
 graph deploy --studio
 ```
 
+=== Contributing
+
+==== Lint
+
+We use https://prettier.io[Prettier] and https:eslint.org[ESLint] for linting code. To run:
+```
+yarn lint
+yarn lint:fix
+yarn format
+yarn format:fix
+```
+
+We encourage to use https://pre-commit.com[pre-commit] Git hooks. After installing pre-commit, to
+install the git hooks, run:
+```
+pre-commit install
+```
```

Comment:
Seems outdated because we switched to husky https://github.com/threshold-network/subgraph/pull/14#discussion_r889897415.

Reply:
Good catch! Done.